core/datainput.gms

```diff
@@ -1207,6 +1208,12 @@ $endif.cm_subsec_model_steel
   p_adj_coeff(ttot,regi,'oae_el')          = 0.8;
   p_adj_coeff(ttot,regi,teGrid)            = 0.3;
   p_adj_coeff(ttot,regi,teStor)            = 0.05;
+  
+  p_adj_coeff(ttot,regi,"elh2")            = 0.75;
```

Comment:
not sure about this - the coal2h/bio2h have lower adjustment costs - I would think that electrolyzers seem a cleaner/more modular/less "heavy chemistry" technology than coal2h/bio2h? 
so I would only go to 0.5 or so. 
but maybe I am overlooking something?

Reply:
Alright. I see the point. I went for coefficient 0.5 (double default) and seed 0.5 (half default). That should be fine. I would not go lower for now as the it needed some increase with the current CAPEX to not be above feasibility benchmarks for 2030, which we saw, for instance, in Ariadne. We might want to reassess after potential CAPEX updates for electrolysis and the BtL/GtL etc. technologies.